docs/chaos-engineering/User Guides/connect-chaos-infrastructures.md

```diff
@@ -0,0 +1,51 @@
+
+## Create Environment
+To add a chaos infrastructure for your target environment, first we need to create a new environment as part of which the chaos infrastructures will exist. Head to the **Environments** page, which lists all the environments added as part of your project. Choose **New Environment**.
+
+![Chaos Environments](./static/connect-chaos-infrastructures/chaos-environments.png)
+
+In the **Create a new environment** modal, provide a name for the environment, optionally you can also add a description and tags. Then select an environment type i.e. **Production** or **Non-Production**. Then select **Create**. This will instantly create a new environment.
+
+![New Environment](./static/connect-chaos-infrastructures/new-environment.png)
+
+## Add Chaos Infrastructure
+Now we can add a chaos infrastructure to our environment. Select **New Chaos Infrastructure**.
+
+You can either setup a chaos infrastructure on existing infrastructures that uses a Harness cluster connector i.e. Harness Kubernetes delegate or install the chaos infrastructure in a new infrastructure.
+
+To use an existing Harness Kubernetes delegate to install your chaos infrastructure, select **On Existing Infrastructures** in the modal. You can use any delegate under the Project, Organization, and Account scope.
+
+![Chaos Infrastructure in Existing Infra](./static/connect-chaos-infrastructures/chaos-infrastructure-in-existing-infra.png)
+
+:::info
+1. A Harness Kubernetes delegate with cluster-wide read/write access can setup both cluster-scoped and namespaced-scoped chaos infrastructures.
+2. A Harness Kubernetes delegate with cluster-wide read access **can't** be used to setup a chaos infrastructures.
+3. A Harness Kubernetes delegate with specific namespace access can only be used to setup a namespace-scoped chaos infrastructure in that specific Kubernetes namespace.
+:::
+
+To install chaos infrastructure on a new infrastructure, select **On New Infrastructures**. Then, select **Continue**.
+Provide a name for the infrastructure. Optionally, you can also provide a description and tags. Then, select **Next**. 
+
+Under the installation mode, choose either **Cluster Wide** or **Namespace Mode** for installing the chaos infrastructure. By default the installation will take place in the `litmus` namespace and uses `litmus` service account, which can be configured under the K8s Cluster Details. Optionally, you can also specify node selectors and K8s tolerations for chaos infrastructure deployment.
```

Comment:
For the brownfield method (using harness Kubernetes connector) if a user has a pre-existing cluster wide installation of the chaos infrastructure on a particular connector, they will no longer be presented with the cluster wide option again (for that specific connector), as cluster scope installation of the chaos infrastructure overrides the existing cluster resources, we may want to add this note here